rust/python_bindings/src/bindings.rs

```diff
@@ -77,4 +77,28 @@ impl Bindings {
     }
 
     ////////////////////////////// Base API //////////////////////////////
+    #[allow(clippy::too_many_arguments)]
+    fn create_collection(
```

Comment:
decided to require Python to pass defaults instead of duplicating the defaults here

Reply:
note to self: align list databases and others